src/plays/web3-staking-dapp/Web3StakingDapp.js

```diff
@@ -0,0 +1,273 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import React, { useState, useEffect } from "react";
+import artifact from "./artifacts/contracts/Staking.sol/Staking.json";
+import { ethers } from "ethers";
+import Navbar from "./components/Navbar";
+import maticLogo from "./images/polygon-matic-logo.svg";
+import { FaRupeeSign } from 'react-icons/fa';
+
+// WARNING: Do not change the entry componenet name
+function Web3StakingDapp(props) {
+
+  // Your Code Start below.
+
+  const ContractAddress = '0x12163B070B97f06F5061D93164D960bbFCfdf965';
+
+  // frontend states
+  const [provider, setProvider] = useState(undefined);
+  const [account, setAccount] = useState(undefined);
+  const [contract, setContract] = useState(undefined);
+  const [accountAddress, setAccountAddress] = useState(undefined);
```

Comment:
We don't need to provide the default value as `undefined` as it will be undefined by default.
```suggestion
  const [provider, setProvider] = useState();
  const [account, setAccount] = useState();
  const [contract, setContract] = useState();
  const [accountAddress, setAccountAddress] = useState();
```